src/pages/patientView/clinicalInformation/PatientViewPageStore.ts

```diff
@@ -173,6 +174,10 @@ export class PatientViewPageStore {
         return fetchMyCancerGenomeData();
     }
 
+    mutationalSignatureData(): IMutationalSignature[]{

```

Comment:
this should be a `remoteData` - look to the others for example, but it should look like 
`readonly mutationalSignatureData = remoteData({
    invoke:async()=>fetchMutationalSignatureData()
})`

later on when you actually call the API you'll be using this format, so better to just do it now